### ISSUE TITLE
[Bug]: Items dont go into network when drawer is not close

### ISSUE THREAD

**Issue by MrStickyPiston** [MattiDragon/ExtendedDrawers#105]
### Description of the issue

Item does not go into storage system (access point) while there is space and connected when not close to drawer. (using vanilla hopper) 

### Reproduction steps

build a drawer system with a drawer a little bit away from a acces point
![2024-04-22_17 45 24](https://github.com/MattiDragon/ExtendedDrawers/assets/97804691/19c4a47e-05c1-4958-b17f-4b62c108526a)
![2024-04-22_17 45 21](https://github.com/MattiDragon/ExtendedDrawers/assets/97804691/1248b4a6-d2fa-408a-b943-b84598183f5b)
![2024-04-22_17 45 05](https://github.com/MattiDragon/ExtendedDrawers/assets/97804691/1bf1665e-0cff-4903-b016-959a9b523a7a)

### Mod version

2.1.1

### Operating System

Kubuntu 22.03

### Minecraft version

1.20.1

### Other mods

file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/artifacts-7.0.3+fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/autoconfig1u-3.4.0.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/BuildersJetpack-1.4.2.jar.disabled
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/cloth-config-6.5.102-fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/continuity-2.0.2+1.18.2.jar.disabled
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/copycats-1.1.1-mc1.18.2-fabric.jar.disabled
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/ctm-fabric-1.0.0+1.18.jar.disabled
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/fabric-api-0.76.0+1.18.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/fabric-api-0.77.0+1.18.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/fabric-language-kotlin-1.10.19+kotlin.1.9.23.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/farmers-delight-fabric-1.18.2-1.2.5.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/jei-1.18.2-fabric-10.2.1.1002.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/unfixed-seeds-1.0.2.jar.disabled
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/appleskin-fabric-mc1.20.1-2.5.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/athena-fabric-1.20.1-3.1.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/balm-fabric-1.20.1-7.2.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/BetterF3-7.0.2-Fabric-1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/cardinal-components-api-5.2.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/chunkloaders-1.2.8a-fabric-mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/fabric-carpet-1.20-1.4.112+v230608.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/appliedenergistics2-fabric-15.1.0.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/cloth-config-11.1.118-fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Clumps-fabric-1.20.1-12.0.0.4.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/createaddition-fabric+1.20.1-1.2.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/cristellib-1.1.5-fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/DisableCustomWorldsAdvice-4.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/entity_model_features_fabric_1.20.1-2.0.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/entity_texture_features_fabric_1.20.1-6.0.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/entityculling-fabric-1.6.2-mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/ExtendedDrawers-2.1.1+mc.1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/extremesoundmuffler-3.41-fabric-1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/fabric-api-0.92.1+1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/ferritecore-6.0.1-fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/fusion-1.1.1-fabric-mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/global-datapack-1.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/farmers-delight-fabric-1.4.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/immersive_paintings-0.6.7+1.20.1-fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/indium-1.0.30+mc1.20.4.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/inventorymanagement-1.3.1+1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/iron-jetpacks-0.4.7.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/JadeAddons-1.20.1-fabric-5.2.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Steam_Rails-1.6.3+fabric-mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/iris-mc1.20.1-1.6.17.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Jade-1.20-fabric-11.8.0.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/jei-1.20.1-fabric-15.3.0.4.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/JustEnoughFarmersRecipes-1.20.1-1.0.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/lambdynamiclights-2.3.2+1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/lazydfu-0.1.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/lithium-fabric-mc1.20.1-0.11.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/modmenu-7.2.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Chipped-fabric-1.20.1-3.0.6.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/libIPN-fabric-1.20-4.0.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/modernfix-fabric-5.16.1+mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Patchouli-1.20.1-84-FABRIC.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/pickable_villagers-1.4.5+1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/rechiseled-1.1.5b-fabric-mc1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/rechiseledcreate-1.0.2-fabric-mc1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/reeses_sodium_options-1.7.2+mc1.20.1-build.101.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/create-fabric-0.5.1-f-build.1417+mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/resourcefullib-fabric-1.20.1-2.1.24.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/scarecrowsterritory-1.1.11-fabric-mc1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/sodium-extra-0.5.4+mc1.20.1-build.115.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/sodium-fabric-0.5.8+mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/starlight-1.1.2+fabric.dbc156f.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/supermartijn642configlib-1.1.8a-fabric-mc1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/toolleveling-fabric-1.20.1-2.0.0.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Hephaestus-1.20.1-3.6.4.273.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/supermartijn642corelib-1.1.17-fabric-mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Terralith_1.20.4_v2.4.11.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/travelersbackpack-fabric-1.20.1-9.1.12.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/waystones-fabric-1.20-14.1.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/graves-3.0.1+1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/mvs-4.1.2-1.20-fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/worldedit-mod-7.2.15.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Xaeros_Minimap_24.1.1_Fabric_1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/XaerosWorldMap_1.38.4_Fabric_1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/yet-another-config-lib-fabric-3.2.2+1.20.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsApi-1.20-Fabric-4.0.4.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterDesertTemples-1.20-Fabric-3.0.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterDungeons-1.20-Fabric-4.0.4.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterEndIsland-1.20-Fabric-2.0.6.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterNetherFortresses-1.20-Fabric-2.0.6.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterMineshafts-1.20-Fabric-4.0.4.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterOceanMonuments-1.20-Fabric-3.0.4.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterStrongholds-1.20-Fabric-4.0.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsBetterWitchHuts-1.20-Fabric-3.0.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/YungsExtras-1.20-Fabric-4.0.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/Zoomify-2.11.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/architectury-9.2.14-fabric.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/artifacts-fabric-9.4.3.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/trinkets-3.7.2.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/WorldEditCUI-1.20+01.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/skinlayers3d-fabric-1.6.2-mc1.20.1.jar
file:///home/sticky/.local/share/PrismLauncher/instances/[1.20.1] Create fabric/.minecraft/mods/dynamic-fps-3.4.2+minecraft-1.20.4-fabric.jar

### Server

Not tested

### Client

Happens

### Logs

https://mclo.gs/YWVcDuq

### Extra Context

many errors in log about drawers.

**Comment by MattiDragon** [comment]
If your drawers are all orthognally touching then accessing all of them should work. Check the following:
1. Is there space in the drawer where you want the item to go?
2. Try running `/graphlib extended_drawers:drawers debugrender start`. Do you see connecting lines between all drawers? If not you can use `/graphlib extended_drawers:drawers updateblocks` to try and fix it.

If you can reliably reporduce the issue in a new world then please explain how

**Comment by MrStickyPiston** [comment]
![2024-04-23_13 32 28](https://github.com/MattiDragon/ExtendedDrawers/assets/97804691/7fe02294-4c71-4648-bfd4-c690b868e623)
it indeed did not connect in the debug render. Nice tool!
![2024-04-23_13 59 18](https://github.com/MattiDragon/ExtendedDrawers/assets/97804691/0511dbcb-c3d6-4f75-8648-5fb09683e05b)
I fixed it all up, most things work now but it seems like create treshold switch detects 100% stack size as 64 items, no matter what upgrades. Should i make a create fabric issue or a extended drawers issue or both?
![2024-04-23_14 02 07](https://github.com/MattiDragon/ExtendedDrawers/assets/97804691/d59cb5c4-5fb9-4a93-b984-2b5268ba13c5)


TLDR: Fixed it for me, new issue with max items in drawer being 64

**Comment by MattiDragon** [comment]
Create threshold switches being weird is almost certainly on their end. If they check the max stack size instead of the capacity they're doing it wrong.

**Comment by MrStickyPiston** [comment]
Seems to work for other storage & in previous versions though

**Comment by MattiDragon** [comment]
Do those other storages have capacities beyond the max stack size? I'm pretty sure that's causing the issue.

**Comment by MrStickyPiston** [comment]
No, but they can store multiple stacks. (Chests, barrels, etc)

**Comment by MrStickyPiston** [comment]
Someone already created an issue for this: https://github.com/Fabricators-of-Create/Create/issues/1459

**Comment by MattiDragon** [comment]
This was completed, right?
